tests/Traits/RenderTrait.php

```diff
@@ -4,11 +4,77 @@
 
 namespace Drupal\Tests\oe_theme\Traits;
 
+use Symfony\Component\DomCrawler\Crawler;
+
 /**
  * Helper rendering trait.
  */
 trait RenderTrait {
 
+  /**
+   * Run various assertion on given HTML string via CSS selectors.
+   *
+   * Specifically:
+   *
+   * - 'count': assert how many times the given HTML elements occur.
+   * - 'equals': assert content of given HTML elements.
+   * - 'contains': assert content contained in given HTML elements.
+   *
+   * Assertions array has to be provided in the following format:
+   *
+   * [
+   *   'count' => [
+   *     '.ecl-page-header' => 1,
+   *   ],
+   *   'equals' => [
+   *     '.ecl-page-header__identity' => 'Digital single market',
+   *   ],
+   *   'contains' => [
+   *     'Digital',
+   *     'single',
+   *     'market',
+   *   ],
+   * ]
+   *
+   * @param string $html
+   *   A render array.
+   * @param array $assertions
+   *   Test assertions.
+   *
+   * @dataProvider renderingDataProvider
```

Comment:
Since we are refactoring this, let's remove this line as it's outdated.

Reply:
Done